### PR TITLE
Indicators comments/textarea label

### PR DESCRIPTION
**Pull request by MirellaAndreoli**
- Label of "Novo Status" text area updated for ""Novo Status": (Indicar as ações no caso de status em amarelo)";
- New css class for the label of "Novo Status" text area;
- Added new class into the mock with an empty list of questions to simulate indicators without questions;

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -317,14 +317,14 @@
...
-                <!--</jvmFlags>-->
-                <!--</configuration>-->
+<!--                <configuration>-->
+<!--                <jvmFlags>-->
+<!--                <jvmFlag>-Xdebug</jvmFlag>-->
+<!--                <jvmFlag>-Xnoagent</jvmFlag>-->
+<!--                <jvmFlag>-Xrunjdwp:transport=dt_socket,address=4001,suspend=y,server=y-->
+<!--                </jvmFlag>-->
+<!--                </jvmFlags>-->
+<!--                </configuration>-->
```
Comment (luanpotter) [inline pom.xml]: why was this change made?

File: src/main/java/br/com/dextra/dexboard/planilha/PlanilhaFactory.java [hunk 2]
```diff
@@ -18,7 +19,10 @@ public static PlanilhaPrincipal principal() {
     }
 
     public static PlanilhaQuestoes questoes(String nomeIndicador) {
-        return (MOCK) ? new MockPlanilhaIndicadoresQuestoes() : new PlanilhaQuestoesImpl(nomeIndicador);
+        if (MOCK) {
+            return nomeIndicador.isEmpty()? new MockPlanilhaIndicadorSemQuestoes() : new MockPlanilhaIndicadoresQuestoes();
```
Comment (luanpotter) [inline src/main/java/br/com/dextra/dexboard/planilha/PlanilhaFactory.java]: add a space before `?`

File: src/main/webapp/css/dx-board.css [hunk 3]
```diff
@@ -470,6 +470,18 @@ div.ui-dialog-titlebar {
   text-shadow: 0 1px 0 #fff;
 }
 
+.label-textarea p{
```
Comment (luanpotter) [inline src/main/webapp/css/dx-board.css]: space before `{`

File: src/main/webapp/css/dx-board.css [hunk 4]
```diff
@@ -470,6 +470,18 @@ div.ui-dialog-titlebar {
...
+  color: #666;
+  font: normal 12px "Open Sans",arial;
```
Comment (luanpotter) [inline src/main/webapp/css/dx-board.css]: space after commas

File: pom.xml (hunk 1 above)
Comment (MirellaAndreoli) [reply pom.xml]: It was necessary to uncomment this configuration to debug the application, and after it, i used the intelliJ's comment short cut that probably had something wrong with it, i will take a look on it.

Comment (luanpotter) [reply pom.xml]: Just curious, I think we can leave it like that if that's intellij default :)

File: src/main/webapp/css/dx-board.css [hunk 5]
```diff
@@ -470,6 +470,18 @@ div.ui-dialog-titlebar {
...
 
+.label-textarea p {
+  color: #666;
+  font: normal 12px "Open Sans", arial;
+  text-shadow: 0 1px 0 #fff;
+}
+
+.label-textarea{
+  color: #666;
+  font: bold 12px "Open Sans",arial;
```
Comment (luanpotter) [inline src/main/webapp/css/dx-board.css]: nit: missing space after comma here

File: src/main/webapp/css/dx-board.css [hunk 6]
```diff
@@ -470,6 +470,18 @@ div.ui-dialog-titlebar {
   text-shadow: 0 1px 0 #fff;
 }
 
+.label-textarea p {
+  color: #666;
+  font: normal 12px "Open Sans", arial;
+  text-shadow: 0 1px 0 #fff;
+}
+
+.label-textarea{
```
Comment (luanpotter) [inline src/main/webapp/css/dx-board.css]: nit: missing space before `{` here

**Pull request merged**